src/pages/integrations/bitpay-card/bitpay-card-topup/bitpay-card-topup.ts

```diff
@@ -567,6 +567,14 @@ export class BitPayCardTopUpPage {
                     );
 
                     return resolve({ amount: newAmountSat, currency: 'sat' });
+                  })
+                  .catch(err => {
+                    throw {
+                      title: this.translate.instant(
+                        'Error in Payment Protocol'
```

Comment:
Can we put something more user-friendly: "Error while fetching the invoice" or something?

Reply:
Yes! done